tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "moduleResolution": "node",
+    "strict": true,
+  }
+}
```

Comment:
(Nit) Could you add a newline to the end of this file? I try to be consistent across the entire project. :)